test/mock/client_mock.py

```diff
@@ -0,0 +1,23 @@
+""" Mock client"""
+from src.constants.constants import PROCESSING_TASKS, CELERY
+
+
+class MockAPIClient:
+    def __init__(self, api_key, wallet_address):
+        self.AUCTION_API = "https://swanhub-cali.swanchain.io"
```

Comment:
We wanna put this API into environment variables?

Reply:
I'm not sure we can do it easily without using any cloud system or .env file